apps/web/src/app/[locale]/tracks/_components/track-enrolled-section.tsx

```diff
@@ -52,17 +32,7 @@ export async function EnrolledTrackSection() {
           </div>
 
           <section className="relative flex w-full flex-row gap-4 overflow-hidden rounded-[2.5rem]">
-            <Carousel>
-              {tracks.map((t) => (
-                <Link
-                  className="group w-[250px] flex-none snap-center focus:outline-none sm:w-[330px] xl:w-[333px]"
-                  href={`/tracks/${t.slug}`}
-                  key={t.id}
-                >
-                  <PersonalTrackCard track={t} />
-                </Link>
-              ))}
-            </Carousel>
+            <EnrolledTrackSectionCarousel tracks={tracks} />
```

Comment:
this is the last problem child.

attempted to pass the track props to this component with no luck